### PR TITLE
[Robot] New test case for updating the pause status and checking the status on RD

### PR DESCRIPTION
**Pull request by rjanjanam**
After creating an open recurring donation using API, The test ensures that after
A pause action is made on the RD and the night job is run, then the
Status of the RD changes to Paused from Active and When the user decides to
Unpause the paused installment then the status on the RD changes back to Active

cci task run robot --org newqaorg  -o suites robot/Cumulus/tests/browser/recurring_donations/zupdate_rd_with_paused_status.robot

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Review by Safiyab — APPROVED** [review]

File: robot/Cumulus/tests/browser/recurring_donations/zupdate_rd_with_paused_status.robot [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    Current Page Should be                    Details                                  npe03__Recurring_Donation__c
+
+    # Get the first installment payment dates to pause
+    ${date1}                                  Get Next Payment Date Number        1     False
+    ${PAUSE_DATES_EDITED}=                    Create List           ${date1}
+    Pause Recurring Donation
+
+    # Verify user is able to click on the pause modal and update with new pause dates
+    # Run the nightly batch job
+    Populate Pause Modal
```
Comment (skristem) [inline robot/Cumulus/tests/browser/recurring_donations/zupdate_rd_with_paused_status.robot:74]: adjust spacing

File: robot/Cumulus/tests/browser/recurring_donations/zupdate_rd_with_paused_status.robot [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+...             robot/Cumulus/resources/RecurringDonationsPageObject.py
+Suite Setup     Run keywords
+...             Enable RD2
+...             Open Test Browser
+...             Setup Test Data
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Keywords ***
+
+Setup Test Data
```
Comment (skristem) [inline robot/Cumulus/tests/browser/recurring_donations/zupdate_rd_with_paused_status.robot:16]: adjust spacing for this keyword to match with test case spacing

**Review by skristem — APPROVED** [review]
looks good, just adjust spacing

**Review by andrewyu-salesforce — APPROVED** [review]
@rjanjanam Looks great

**Comment by rponti-sforg** [comment]
Will be merging this later today after builds have pased.

**Pull request merged**